api/views.py

```diff
@@ -17,11 +20,24 @@ class DocumentDetail(generics.RetrieveAPIView):
 def get_document_blocks(request, pk):
     blocks = Block.objects.filter(block_document=pk)
     serializer = BlockSerializer(blocks, many=True)
-    return rest_framework.response.Response(serializer.data, status=200)
+    return Response(serializer.data, status=200)
```

Comment:
I think there should be a check if the serialised is correct incase the document requested doesn't exist